backend/controllers/UserController.go

```diff
@@ -0,0 +1,87 @@
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/fourtf/studyhub/models"
+	"github.com/fourtf/studyhub/utils"
+	"golang.org/x/crypto/bcrypt"
+)
+
+var db = utils.ConnectToDB()
```

Comment:
Consider doing this in `main.go` so a) the order of execution in clear and b) you see all connections that get created in the main